guides/source/action_controller_overview.md

```diff
@@ -1073,10 +1073,10 @@ class ClientsController < ApplicationController
 end
 ```
 
-For this example to work, you have to add the PDF MIME type to Rails. This can be done by adding the following line to the file `config/initializers/mime_types.rb`:
+This example works because [Rails already registers the `"application/pdf"` MIME type](https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/http/mime_types.rb) at startup (alongside other common types). If you need additional MIME types, call [`Mime::Type.register`](https://api.rubyonrails.org/classes/Mime/Type.html#method-c-register) in the file `config/initializers/mime_types.rb`. For example:
```

Comment:
We could probably remove this content altogether, as it feels out of place now, also rare to do so.

There is no documentation on https://edgeapi.rubyonrails.org/classes/Mime/Type.html#method-c-register , that is a better place to start adding this instead.

Reply:
I still think it's important to have that comment in there, because it can be surprising to someone seeing the `respond_to` followed by `format.pdf` and wondering "how does Rails know what to do with PDF files?", and "where does the `.pdf` method name comes from?" or even "what other formats does it support?". Those are the questions I had every time I read that part of the guides.